### PR TITLE
Bug fixes and CONTRIBUTING.md improvements

### PR DESCRIPTION
**Pull request by gruyaume**

### CODE REVIEW

**Review by Gmerold — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,6 +12,23 @@ The charms and bundles in this repository are specifically developed for the
...
-## Contributing to a magma component
-Each component has its own `CONTRIBURING.md` file that will detail how you can test, deploy and
-publish this specific component. Please refer to that file.
+## Code contributions
+If you want to propose a new feature, a bug fix or a documentation improvement:
+- Create a new branch from main.
+- Commit and push your changes to this branch.
+- Validate that all Github actions pass.
+- Create a pull request in [github](https://github.com/canonical/charmed-magma/pulls).
+- Your pull request will be reviewed by one of the repository maintainer.
```
Comment (sanchezfdezjavier) [inline CONTRIBUTING.md]: "s" lacking?
```suggestion
- Your pull request will be reviewed by one of the repository maintainers.
```

**Review by sanchezfdezjavier — APPROVED** [review]

File: orchestrator-bundle/orc8r-service-registry-operator/src/charm.py [hunk 2]
```diff
@@ -73,6 +79,15 @@ def _configure_orc8r(self, event):
...
 
+    def _namespace(self) -> str:
+        """The Kubernetes namespace we're running in.
+
+        Returns:
+            str: A string containing the name of the current Kubernetes namespace.
+        """
+        with open("/var/run/secrets/kubernetes.io/serviceaccount/namespace", "r") as f:
+            return f.read().strip()
+
```
Comment (Gmerold) [inline orchestrator-bundle/orc8r-service-registry-operator/src/charm.py]: In other charms we just use `self._namespace = self.model.name` in the constructor. Seems simpler ;)

File: CONTRIBUTING.md (hunk 1 above)
Comment (gruyaume) [reply CONTRIBUTING.md]: Fixed

File: orchestrator-bundle/orc8r-service-registry-operator/src/charm.py (hunk 2 above)
Comment (gruyaume) [reply orchestrator-bundle/orc8r-service-registry-operator/src/charm.py]: Fixed

**Review by sanchezfdezjavier — APPROVED** [review]

**Pull request merged**